src/components/withDelayToggleButtonState.js

```diff
@@ -0,0 +1,80 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import getComponentDisplayName from '../libs/getComponentDisplayName';
+
+const withDelayToggleButtonStatePropTypes = {
+    /** A value whether the button state is complete */
+    isButtonStateComplete: PropTypes.bool.isRequired,
```

Comment:
NAB:  How about this?

`isButtonStateComplete` -> `isDelayButtonStateComplete`

Reply:
Yup, that’s more logical!